### PR TITLE
Update manifest for Everything 1.4.1.1018

### PR DESCRIPTION
**Pull request by voidtools**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/70586)

### CODE REVIEW

**Comment by voidtools** [comment]
uploaded installer to website

File: manifests/v/voidtools/Everything/1.4.1.1018/voidtools.Everything.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+UpgradeBehavior: install
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://www.voidtools.com/Everything-1.4.1.1018.x64.msi
+  InstallerSha256: 754A823B3A570CD8C1DD5D27AED694B056EAEFB5C5AE32A23029250DC728423D
+  AppsAndFeaturesEntries:
+  - DisplayName: Everything 1.4.1.1018 (x64)
+    DisplayVersion: 1.4.1.1018
```
Comment (Trenly) [inline manifests/v/voidtools/Everything/1.4.1.1018/voidtools.Everything.installer.yaml:25]:
```suggestion
```

File: manifests/v/voidtools/Everything/1.4.1.1018/voidtools.Everything.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: msi
+  InstallerUrl: https://www.voidtools.com/Everything-1.4.1.1018.x86.msi
+  InstallerSha256: 80E2DD7FC1D264106F5125FA6B1D62ADCD1AAD54590DCA2EC60AE9F3BFBC9CC1
+  AppsAndFeaturesEntries:
+  - DisplayName: Everything 1.4.1.1018 (x86)
+    DisplayVersion: 1.4.1.1018
```
Comment (Trenly) [inline manifests/v/voidtools/Everything/1.4.1.1018/voidtools.Everything.installer.yaml:33]:
```suggestion
```

File: manifests/v/voidtools/Everything/1.4.1.1018/voidtools.Everything.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://www.voidtools.com/Everything-1.4.1.1018.x64-Setup.exe
+  InstallerSha256: A387EB0C906E2243EB125D5B1091DF3AB6D0183A6BA7DAD9BF10EF0037B0CFB6
+  AppsAndFeaturesEntries:
+    - DisplayName: Everything 1.4.1.1018 (x64)
+      DisplayVersion: 1.4.1.1018
```
Comment (Trenly) [inline manifests/v/voidtools/Everything/1.4.1.1018/voidtools.Everything.installer.yaml:40]:
```suggestion
```

File: manifests/v/voidtools/Everything/1.4.1.1018/voidtools.Everything.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://www.voidtools.com/Everything-1.4.1.1018.x86-Setup.exe
+  InstallerSha256: 0E822C0775C8CFE29121109D8378E10BD8D7FF0C93C61383D5FFABBDD8A25B18
+  AppsAndFeaturesEntries:
+  - DisplayName: Everything 1.4.1.1018 (x86)
+    DisplayVersion: 1.4.1.1018
```
Comment (Trenly) [inline manifests/v/voidtools/Everything/1.4.1.1018/voidtools.Everything.installer.yaml:47]:
```suggestion
```

File: manifests/v/voidtools/Everything/1.4.1.1018/voidtools.Everything.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://www.voidtools.com/Everything-1.4.1.1018.x64.en-US-Setup.exe
+  InstallerSha256: A7F79D6E05314563422E139AC4C099BAFB1C671FF4E1C738D8B3A4749E1AFC82
+  AppsAndFeaturesEntries:
+    - DisplayName: Everything 1.4.1.1018 (x64)
+      DisplayVersion: 1.4.1.1018
```
Comment (Trenly) [inline manifests/v/voidtools/Everything/1.4.1.1018/voidtools.Everything.installer.yaml:55]:
```suggestion
```

File: manifests/v/voidtools/Everything/1.4.1.1018/voidtools.Everything.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://www.voidtools.com/Everything-1.4.1.1018.x86.en-US-Setup.exe
+  InstallerSha256: DDCC5480D3BC59FF70F786A6D3D32DA992094DC15E7A5BDB1D5EF875010B9832
+  AppsAndFeaturesEntries:
+  - DisplayName: Everything 1.4.1.1018 (x86)
+    DisplayVersion: 1.4.1.1018
```
Comment (Trenly) [inline manifests/v/voidtools/Everything/1.4.1.1018/voidtools.Everything.installer.yaml:63]:
```suggestion
```

File: manifests/v/voidtools/Everything/1.4.1.1018/voidtools.Everything.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: /NORESTART
+UpgradeBehavior: install
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
```
Comment (Trenly) [inline manifests/v/voidtools/Everything/1.4.1.1018/voidtools.Everything.installer.yaml:20]:
```suggestion
  InstallerType: wix
```

File: manifests/v/voidtools/Everything/1.4.1.1018/voidtools.Everything.installer.yaml [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+  InstallerUrl: https://www.voidtools.com/Everything-1.4.1.1018.x64.msi
+  InstallerSha256: 754A823B3A570CD8C1DD5D27AED694B056EAEFB5C5AE32A23029250DC728423D
+  AppsAndFeaturesEntries:
+  - DisplayName: Everything 1.4.1.1018 (x64)
+    DisplayVersion: 1.4.1.1018
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: msi
```
Comment (Trenly) [inline manifests/v/voidtools/Everything/1.4.1.1018/voidtools.Everything.installer.yaml:28]:
```suggestion
  InstallerType: wix
```

**Review by Trenly — CHANGES_REQUESTED** [review]
When the DisplayVersion is the same as PackageVersion, it should not be included in the AppsAndFeaturesEntries as it will cause additional version mapping on the backend which could lead to increased processing and loading times

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**